site/components/DateSelector.test.tsx

```diff
@@ -58,6 +92,11 @@ describe("DateSelector", () => {
                     inputId={"publish-end-date"}
                     inputName={"publishStartDateDay"}
                     disabled
+                    pageState={blankInputs}
+                    // eslint-disable-next-line @typescript-eslint/no-empty-function
+                    updatePageState={() => {}}
+                    // eslint-disable-next-line @typescript-eslint/no-empty-function
+                    updaterFunction={() => {}}
```

Comment:
would a file disable of this rule be more appropriate?

Reply:
I dont really have much of an opinion - @lj-kpmg ?

Reply:
Yeah, put it as a file level disable if it's being used in a few places

Reply:
disabled it for the file